contracts/hyperlane-mailbox/tests/harness.rs

```diff
@@ -294,117 +287,6 @@ async fn test_latest_checkpoint() {
-        .with_account(initial_owner_wallet.clone())
-        .unwrap()
-        .methods()
-        .transfer_ownership(new_owner.clone())
-        .tx_params(TxParameters::default())
-        .call()
-        .await
-        .unwrap();
-
-    // Ensure the owner is now the new owner
-    let owner = mailbox.methods().owner().simulate().await.unwrap().value;
-    assert_eq!(owner, new_owner);
-
-    // Ensure an event about the ownership transfer was logged
-    let ownership_transferred_events = transfer_ownership_call
-        .get_logs_with_type::<OwnershipTransferredEvent>()
-        .unwrap();
-    assert_eq!(
-        ownership_transferred_events,
-        vec![OwnershipTransferredEvent {
-            previous_owner: initial_owner,
-            new_owner: new_owner.clone(),
-        }]
-    );
-
-    // Ensure the old owner can't transfer ownership anymore
-    let invalid_transfer_ownership_call = mailbox
-        .with_account(initial_owner_wallet)
-        .unwrap()
-        .methods()
-        .transfer_ownership(new_owner.clone())
-        .tx_params(TxParameters::default())
-        .call()
-        .await;
-    assert!(invalid_transfer_ownership_call.is_err());
-    assert_eq!(
-        get_revert_string(invalid_transfer_ownership_call.err().unwrap()),
-        "!owner"
-    );
-}
-
-#[tokio::test]
-async fn test_transfer_ownership_to_some() {
-    let (mailbox, _, _, _) = get_contract_instance().await;
-
-    // The current owner before the transfer / old owner after the transfer
-    let initial_owner: Option<Identity> = Some(Identity::Address(
-        Address::from_str(INITIAL_OWNER_ADDRESS).unwrap(),
-    ));
-    let new_owner: Option<Identity> = Some(Identity::Address(
-        Address::from_str("0xcafecafecafecafecafecafecafecafecafecafecafecafecafecafecafecafe")
-            .unwrap(),
-    ));
-
-    transfer_ownership_test_helper(&mailbox, initial_owner, new_owner).await;
-}
-
-#[tokio::test]
-async fn test_transfer_ownership_to_none() {
-    let (mailbox, _, _, _) = get_contract_instance().await;
-
-    // The current owner before the transfer / old owner after the transfer
-    let initial_owner: Option<Identity> = Some(Identity::Address(
-        Address::from_str(INITIAL_OWNER_ADDRESS).unwrap(),
-    ));
-    let new_owner: Option<Identity> = None;
-
-    transfer_ownership_test_helper(&mailbox, initial_owner, new_owner).await;
-}
-
-#[tokio::test]
-async fn test_transfer_ownership_reverts_if_not_owner() {
-    let (mailbox, _, _, _) = get_contract_instance().await;
-    // The default wallet in Mailbox is not the owner
-    let invalid_transfer_ownership_call = mailbox
-        .methods()
-        .transfer_ownership(None)
-        .tx_params(TxParameters::default())
-        .call()
-        .await;
-    assert!(invalid_transfer_ownership_call.is_err());
-    assert_eq!(
-        get_revert_string(invalid_transfer_ownership_call.err().unwrap()),
-        "!owner"
-    );
-}
-
 #[tokio::test]
```

Comment:
Just realizing that we don't have a test case to make sure that only the owner can set the default ISM, created https://github.com/hyperlane-xyz/fuel-contracts/issues/82 to track that

Reply:
Ended up just adding this to https://github.com/hyperlane-xyz/fuel-contracts/pull/75